cypress/e2e/inventory/search/browse-contributors-linked.cy.js

```diff
@@ -0,0 +1,129 @@
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import BrowseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+
+describe('Browse Contributors in Inventory (linked records)', () => {
+  const testData = {
+    contributorName: 'Lee, Stanley Test C359596',
+  };
+
+  const marcFiles = [
+    {
+      marc: 'marcBibC359596.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthC359596_1.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      naturalId: 'n831692600011181',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthC359596_2.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      naturalId: 'n831692600011182',
+      numOfRecords: 1,
+    },
+  ];
+
+  const tagInfo = [
+    {
+      rowIndex: 12,
+      authNaturalId: 'n831692600011181',
+    },
+    {
+      rowIndex: 13,
+      authNaturalId: 'n831692600011182',
+    }
+  ];
+
+  const createdRecordIDs = [];
+
+  before('Creating data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+    ]).then(createdUserProperties => {
+      testData.userProperties = createdUserProperties;
+
+      marcFiles.forEach(marcFile => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+          DataImport.uploadFile(marcFile.marc, marcFile.fileName);
+          JobProfiles.waitLoadingList();
+          JobProfiles.searchJobProfileForImport(marcFile.jobProfileToRun);
+          JobProfiles.runImportFile();
+          JobProfiles.waitFileIsImported(marcFile.fileName);
+          Logs.checkStatusOfJobProfile('Completed');
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then(link => {
+              createdRecordIDs.push(link.split('/')[5]);
+            });
+          }
+        });
+      });
+
+      cy.loginAsAdmin({ path: TopMenu.inventoryPath, waiter: InventoryInstances.waitContentLoading }).then(() => {
```

Comment:
You are already login as Admin

Reply:
Replaced `cy.loginAsAdmin()` with `cy.visit()` and `InventoryInstances.waitContentLoading()`